rnndb/graph/g80_texture.xml

```diff
@@ -6,157 +6,312 @@ xsi:schemaLocation="http://nouveau.freedesktop.org/ rules-ng.xsd">
 
 <import file="nvchipsets.xml"/>
 <import file="g80_defs.xml"/>
+<import file="nv_defs.xml"/>
```

Comment:
rnn will complain about inv_memory_layout being an unknown type without this. But this was already the case before this change, so let me move this into its own patch.